charts/gardener-extension-shoot-networking-filter/templates/poddisruptionbudget.yaml

```diff
@@ -17,6 +17,4 @@ spec:
     matchLabels:
       app.kubernetes.io/name: gardener-extension-{{ .Values.serviceName }}
       app.kubernetes.io/instance: {{ .Release.Name }}
-{{- if semverCompare ">= 1.26-0" .Capabilities.KubeVersion.GitVersion }}
-  unhealthyPodEvictionPolicy: AlwaysAllow
-{{- end }}
+  unhealthyPodEvictionPolicy: AlwaysAllow
```

Comment:
You could preserve the empty line at the end of the file.
`make generate` has to be executed after this change.